docs/userguide/dockerimages.md

```diff
@@ -210,23 +193,23 @@ command.
     0b2616b0e5a8 ouruser/sinatra:v2
     4f177bd27a9ff0f6dc2a830403925b5360bfe0b93d476f7fc3231110e7f71b1c
 
-Here we've used the `docker commit` command. We've specified two flags: `-m`
+Here you've used the `docker commit` command. You've specified two flags: `-m`
```

Comment:
just notice that this document uses a lot of `'ve`; do we have a preference here? (I was taught that in writing, it's prefered to use the full form; "you have" in stead of "you've"